ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -388,6 +458,7 @@ namespace ngraph
         void InputModelPDPD::InputModelPDPDImpl::extractSubgraph(
             const std::vector<Place::Ptr>& inputs, const std::vector<Place::Ptr>& outputs)
         {
+            m_graph_changed = true;
```

Comment:
may not be needed here as `overrideAllInputs/overrideAllOutputs` already set this internally

Reply:
True, but what if those methods get changed? I don't think it is a big problem to have this flag set here too.